From 6227e0a02332cbf91d53b2dc84371eae35dea264 Mon Sep 17 00:00:00 2001
From: Michael Pauzner <mpauzner@habana.ai>
Date: Thu, 12 Dec 2024 16:10:56 +0200
Subject: [PATCH] [SW-207648] nic: clean-up nic ena set api

Remove old API.
Step 2, remove temporary #define after multi-repository merge.

Change-Id: Icadb985c9d1c34c38005642d16937c1bafd4ef2a
Signed-off-by: Michael Pauzner <mpauzner@habana.ai>
---

diff --git a/include/baremetal.h b/include/baremetal.h
index c7fd4b2..f6f07c4 100644
--- a/include/baremetal.h
+++ b/include/baremetal.h
@@ -202,7 +202,6 @@
 void preboot_pcie_enable(void);
 void preboot_pcie_exec_hw_wa(void);
 int preboot_set_nic_en(u32 nic_en);
-#define preboot_nic_en(nic_en)		preboot_set_nic_en(nic_en)
 bool preboot_host_gpio_asserted(void);
 void preboot_sp_sram_init(void);
 void preboot_cond_arc0_reset(void);
